docs/quickstart.md

```diff
@@ -67,12 +143,11 @@ import nemos as nmo
 
 # initialize a Gamma GLM with Ridge regularization
 model = nmo.glm.GLM(
-    regularizer="ridge", 
+    regularizer="Ridge", 
     observation_model=nmo.observation_models.GammaObservations()
 )
 ```
 
-
 ### Pre-processing with `pynapple`
```

Comment:
... and I would keep the remaining in the Quickstart but maybe with an extra title like "Compatibility with other packages". But not sure. This could be a page on its own but not sure where it would fit.